tests/test_run.py

```diff
@@ -42,3 +43,10 @@ def test_run_script_from_internet(pipx_temp_env, capsys):
             "6bdfbb6e9c1132b1c38fdd2f195d4a24c540c324/pipx-demo.py",
         ]
     )
+
+def test_run_ensure_null_pythonpath():
+    assert 'None' in subprocess.run(
+        [sys.executable, "-m", "pipx", "run", "ipython", "-c", "import os; print(os.environ.get('PYTHONPATH'))"],
+        universal_newlines=True,
+        env={'PYTHONPATH': 'test'},
```

Comment:
To fix the [windows test run](https://travis-ci.org/pipxproject/pipx/jobs/603878395#L110), you'll need to update the existing environment here.
https://github.com/mgedmin/check-manifest/commit/90b89611cd1d28d15748825dac03f8871a6a4c57 :eyes: